### PR TITLE
[REF] rename beesdoo_pos_coop_status -> pos_coop_status

### LINKED ISSUE #340: Rename `beesdoo_pos_coop_status` to `pos_cooperator_status`

### PR DESCRIPTION
**Pull request by victor-champonnois**
## Description
fix https://github.com/beescoop/Obeesdoo/issues/340

## Checklist before approval

- [ ] Tests are present (or not needed).
- [ ] Credits/copyright have been changed correctly.
- [ ] Change log snippet is present.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

**Review by robinkeunen — CHANGES_REQUESTED** [review]
This module has nothing to do with cooperatives anymore. Would `pos_partner_can_shop` be a better name ?

What do you think @remytms 

Sorry for duplicate work.

**Comment by robinkeunen** [comment]
@remytms @victor-champonnois or `pos_shift_partner_can_shop` ?

File: pos_shift_partner_can_shop/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+{
+    "name": "Point of sale - Cooperator Status",
+    "summary": """POS Support for cooperator status.""",
```
Comment (victor-champonnois) [inline pos_shift_partner_can_shop/__manifest__.py]: needs changing

File: pos_shift_partner_can_shop/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+This module displays in the pos whether the cooperator can shop or not.
+If the customer / cooperator cannot shop or is not set, a confirm popup
+is shown.
```
Comment (victor-champonnois) [inline pos_shift_partner_can_shop/readme/DESCRIPTION.rst]: Needs changing

**Review by victor-champonnois — COMMENTED** [review]
coop_status.css needs to be renamed

**Comment by victor-champonnois** [comment]
`pos_shift_partner_can_shop` seems more precise, because it depends on the shift modules

File: pos_shift_partner_can_shop/readme/DESCRIPTION.rst (hunk 2 above)
Comment (robinkeunen) [reply pos_shift_partner_can_shop/readme/DESCRIPTION.rst]: I think this is correct 🤔

File: pos_shift_partner_can_shop/__manifest__.py (hunk 1 above)
Comment (robinkeunen) [reply pos_shift_partner_can_shop/__manifest__.py]: fixed

File: pos_shift_partner_can_shop/readme/DESCRIPTION.rst (hunk 2 above)
Comment (victor-champonnois) [reply pos_shift_partner_can_shop/readme/DESCRIPTION.rst]: It's just that it's mentionning cooperators, and we can have shifts without cooperators no ?

Comment (robinkeunen) [reply pos_shift_partner_can_shop/readme/DESCRIPTION.rst]: Ha yes, thank you 👍

**Review by robinkeunen — APPROVED** [review]

**Pull request merged**